tox.ini

```diff
@@ -12,6 +12,13 @@ covercmd =
       --cov=unit_tests \
       --cov-config {toxinidir}/.coveragerc \
       unit_tests
+    py.test --quiet \
+      --cov=google.cloud \
+      --cov=unit_tests \
+      --cov-append \
+      --cov-config {toxinidir}/.coveragerc \
+      core/unit_tests
+    coverage report --show-missing --fail-under=100
```

Comment:
Yup. Planning on writing a script to handle it long-term. Wanted to keep this PR as small as possible though so left out the script.